packages/catalog-client/src/generated/index.ts

```diff
@@ -15,3 +15,4 @@
  */
 
 export * from './apis';
+export * from './models';
```

Comment:
Not sure what the right approach is here, this is just an internal change due to updates to the `generate-client` command. Nothing was _actually_ updated in catalog-client, does this need a changeset?